### PR TITLE
Add bitcoin regtest support

### PR DESCRIPTION
**Pull request by afk11**
Adds bitcoin_regtest.json, a copy of bitcoin_regtest.json,
with modified coin_name, coin_shortcut, coin_label,
hash_genesis_block, bech32_prefix.

Fixes https://github.com/trezor/trezor-common/issues/270

### CODE REVIEW

**Comment by afk11** [comment]
I'm not sure if I should change the blockbook value, do you have a suggestion @prusnak @matejcik?

**Comment by matejcik** [comment]
yes, please make it an empty list.

**Comment by afk11** [comment]
thanks, forced pushed

File: common/defs/bitcoin/bitcoin_regtest.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "coin_name": "Regtest",
+  "coin_shortcut": "REGTEST",
+  "coin_label": "Regtest",
+  "website": "https://bitcoin.org",
+  "github": "https://github.com/bitcoin/bitcoin",
+  "maintainer": "Pavol Rusnak <stick@satoshilabs.com>",
```
Comment (prusnak) [inline common/defs/bitcoin/bitcoin_regtest.json]: please change the maintainer - you are submitting the file and you are responsible for its contents

File: common/defs/bitcoin/bitcoin_regtest.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "Normal": 10
+  },
+  "dust_limit": 546,
+  "blocktime_seconds": 600,
+  "uri_prefix": "bitcoin",
+  "min_address_length": 27,
+  "max_address_length": 34,
+  "bitcore": [],
+  "blockbook": [
+    "https://tbtc1.trezor.io",
```
Comment (prusnak) [inline common/defs/bitcoin/bitcoin_regtest.json]: Remove these two.

File: common/defs/bitcoin/bitcoin_testnet.json [hunk 3]
```diff
@@ -33,10 +33,7 @@
   "min_address_length": 27,
   "max_address_length": 34,
   "bitcore": [],
-  "blockbook": [
-    "https://tbtc1.trezor.io",
-    "https://tbtc2.trezor.io"
-  ],
+  "blockbook": [],
```
[the remark this answers is not in the document]
Comment (afk11) [reply common/defs/bitcoin/bitcoin_testnet.json]: oops, thanks

File: common/defs/bitcoin/bitcoin_regtest.json (hunk 1 above)
Comment (afk11) [reply common/defs/bitcoin/bitcoin_regtest.json]: done

File: common/defs/bitcoin/bitcoin_regtest.json (hunk 2 above)
Comment (afk11) [reply common/defs/bitcoin/bitcoin_regtest.json]: done

**Pull request merged**